app/views/finance/ecf/statements/show.html.erb

```diff
@@ -2,20 +2,18 @@
 <% content_for :before_content, govuk_back_link(text: "Back", href: select_provider_ecf_finance_payment_breakdowns_path) %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <ul class="govuk-list">
-      <% @statements.each do |statement| %>
-        <li>
-          <%= text_otherwise_link_to(statement_display_text(statement), finance_ecf_payment_breakdown_statement_path(@ecf_lead_provider, statement), statement == @statement) %>
-        </li>
-      <% end %>
-    </ul>
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-xl">Early career framework (ECF)</h1>
+
```

Comment:
Could add heading to en.yml

Reply:
I've extracted the h1 copy to an i18n string